src/app/app.html

```diff
@@ -26,6 +26,6 @@
     </ion-list>
   </ion-content>
 </ion-menu>
-<ion-nav id="myNav" #myNav [root]="rootPage"></ion-nav>
+<ion-nav id="myNav" #myNav [root]="rootPage" ></ion-nav>
 <!-- Icon for manoeuvre modal hidden by default -->
-<ion-icon name="arrow-down" color="light" id="modal-chevron"></ion-icon>
+<ion-icon name="arrow-down" color="light" id="modal-chevron"></ion-icon>
```

Comment:
Don't need to delete the blank line at the end of the file, just means your name is in git history despite you not actually changing anything important in this file